mycroft/client/wifisetup/main.py

```diff
@@ -38,6 +42,29 @@
 
 LOG = getLogger("WiFiClient")
 
+SCRIPT_DIR = dirname(realpath(__file__))
+
+
+def var_dump(var, prefix=''):
+    my_type = '[' + var.__class__.__name__ + '(' + str(len(var)) + ')]:'
+    print(prefix, my_type)
+    prefix += '    '
+    for i in var:
+        if type(i) in (list, tuple, dict, set):
+            var_dump(i, prefix)
+        else:
+            if isinstance(var, dict):
+                print(prefix, i, ': (', var[i].__class__.__name__, ')', var[i])
+            else:
+                print(prefix, '(', i.__class__.__name__, ')', i)
+
+
+def cli_no_output(*args):
```

Comment:
I respectfully disagree about the comments.  The mycroft-core code has virtually no comments besides the boilerplate copyright at the top, and that is not a good thing.  Comments are important to convey intent so subsequent programmers can tell what the code is *supposed* to be doing when there is a bug and it doesn't actually do something correctly.  Check out the source code for Linux or any other major project sometime, it is chock full of comments.